node/src/protocol.rs

```diff
@@ -121,15 +121,15 @@ impl Debug for Message {
             Message::GetRequest { tag, serialized_id } => f
                 .debug_struct("GetRequest")
                 .field("tag", tag)
-                .field("serialized_item", &HexFmt(serialized_id))
+                .field("serialized_item", &base16::encode_lower(serialized_id))
```

Comment:
```suggestion
                .field("serialized_item", &HexFmt(serialized_id))
```

Reply:
Fixed in [e93dc37](https://github.com/nilz3ro/casper-node/pull/5/commits/e93dc3787a9983c352ebfdba0fc6bccb8f4d7c41).